CONTRIBUTING.md

```diff
@@ -3,29 +3,35 @@
 We'd love to accept your patches and contributions to this project. There are
 just a few guidelines you need to follow.
 
-
 ## General Guidelines
 
 1. The primary focus of this repo is to provide introductory Apigee samples of low to medium complexity, aimed at developers who are new to Apigee
     - More complex projects may be better suited for the [Apigee DevRel](https://github.com/apigee/devrel) repo
-1. Projects in this repo are targeted for [Apigee X](https://cloud.google.com/apigee/docs) and [hybrid](https://cloud.google.com/apigee/docs/hybrid/latest/what-is-hybrid). We do not accept samples for the Apigee Edge platform (these may be found [here](https://github.com/apigee/api-platform-samples))
+2. Projects in this repo are targeted for [Apigee X](https://cloud.google.com/apigee/docs) and [hybrid](https://cloud.google.com/apigee/docs/hybrid/latest/what-is-hybrid). We do not accept samples for the Apigee Edge platform (these may be found [here](https://github.com/apigee/api-platform-samples))
     - More information about the different versions of Apigee can be found [here](https://cloud.google.com/apigee/docs/api-platform/get-started/compare-apigee-products)
```

Comment:
While we're here we should update the link to https://docs.apigee.com/migration-to-x/compare-apigee-edge-to-apigee-x